cloudfoundry-client/src/main/java/org/cloudfoundry/client/v3/spacequotadefinitions/_Apps.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2013-2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.cloudfoundry.client.v3.spacequotadefinitions;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import org.cloudfoundry.Nullable;
+import org.immutables.value.Value;
+
+/**
+ * Quotas that affect applications and application sub-resources
+ */
+@JsonDeserialize
+@Value.Immutable
+abstract class _Apps {
```

Comment:
So `Apps`, but also `Routes` and `Services` are exactly the same for org quotas and space quotas.

Maybe those objects should be shared? I'm thinking something like the following package structure:

```
- v3/
   -  quotas/
      -  _Apps
      - _Routes
      - _Services
        - spaces/
        - organizations/
```

What do you think?